migrations.go

```diff
@@ -103,6 +103,16 @@ func migrate_8_AnnounceDefault(cfg *Config, beforeV1B2 bool) bool {
 	return false
 }
 
+func migrate_9_WalletDomain(cfg *Config) bool {
+	if len(cfg.Services.ExchangeDomain) == 0 {
+		ds := DefaultServicesConfig()
```

Comment:
Need a small check here: if the other domains are on testnet domain it should migrate to `DefaultServicesConfigTestnet()`

Reply:
fixed